style.css

```diff
@@ -0,0 +1,269 @@
+* {
+	box-sizing:	border-box;
+	font-family: "Arial";
+	font-size: 15px;
+	background-color:#000000;
+	letter-spacing: 2px;
+}
+/*TAGS*/
+nav {
+	position: fixed;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	background-color: rgba(0,0,0,1.0);
+	width: 100%;
+	margin-left: 0;
+	top: 0;
+	left: 0;
+	z-index: 1;
```

Comment:
for z-index it is recommended to increment by 10 each time, e.g. if the navbar should be in front of the rest, we give it z-index: 10. this is because if we ever want to slot another layer in between, we can do so without changing the navbar to z-index-2.